### ISSUE TITLE
JFrog Account seems inactive

### ISSUE THREAD

**Issue by finalgamer** [sobolevn/git-secret#1013]
I am usually download git-secret from https://gitsecret.jfrog.io/artifactory/git-secret-apk/latest-stable/main on my alpine based CI/CD job.
Since today it can not install the package. Visiting the page in my browser shows a prompt to reactivate the account. I assume something expired on your side.

Reproduce on alpine 3.7
```bash
sh -c "echo 'https://gitsecret.jfrog.io/artifactory/git-secret-apk/all/main'" >> /etc/apk/repositories
wget -O /etc/apk/keys/git-secret-apk.rsa.pub 'https://gitsecret.jfrog.io/artifactory/api/security/keypair/public/repositories/git-secret-apk'
apk add --update --no-cache git-secret

# Testing, that it worked:
git secret --version

```

**Comment by pgaxatte** [comment]
Same issue here with the debian key for the repository https://gitsecret.jfrog.io/artifactory/git-secret-deb

https://gitsecret.jfrog.io/artifactory/api/gpg/key/public is not available either

**Comment by dkonopka** [comment]
same story for me, is it a change that will be fixed today?

**Comment by pgaxatte** [comment]
Meanwhile I'm using something like this on Debian/Ubuntu:

```
GIT_SECRET_VERSION=0.5.0
tmpdir="$(mktemp -d)"
trap "rm -rf '${tmpdir}'" EXIT
curl -sfLo - "https://github.com/sobolevn/git-secret/archive/refs/tags/v${GIT_SECRET_VERSION}.tar.gz" \
  | tar --strip-components=1 -C "${tmpdir}" -xzf -
make -C "${tmpdir}" build
sudo make -C "${tmpdir}" install
```

You just need `make` since `git-secret` is just a bunch of concatenated shell scripts

**Comment by sobolevn** [comment]
Working on it

**Comment by David-Klemenc** [comment]
I put this in my CI/CD script (on Debian):

```sh
    - echo "Installing git secret"
    - apt-get update
    - apt-get -y install gawk
    - git clone https://github.com/sobolevn/git-secret.git git-secret
    - cd git-secret && make build
    - PREFIX="/usr/local" make install
    - cd ..
  ```

**Comment by dkonopka** [comment]
it looks that jfrog package is working right now properly, thx @sobolevn :)

maybe its good idea to add an alternative solution when jfrog is down to the documentation.

**Comment by sobolevn** [comment]
I think that we already have `Manual Installation` case in the docs 🤔 

I am happy that it now works, I just clicked some buttons to restore the repo.
The question is: why do they archive repos that have existing users? 😮

**Comment by dkonopka** [comment]
And.... issue is still alive @sobolevn :( 

> JFrog's Free-Tier subscription has been deactivated.

**Comment by marcossv9** [comment]
It is still happening for us right now:
![image](https://github.com/sobolevn/git-secret/assets/12616390/15fe264f-41c0-4242-bc87-54a01bad0e5a)

```
+ wget https://gitsecret.jfrog.io/artifactory/git-secret-apk/latest-stable/main/x86_64/git-secret-0.5.0.apk; apk add --no-cache --allow-untrusted git-secret-0.5.0.apk; rm git-secret-0.5.0.apk;
Connecting to gitsecret.jfrog.io (3.234.157.227:443)
Connecting to landing.jfrog.com (52.204.61.45:443)
saving to 'git-secret-0.5.0.apk'
git-secret-0.5.0.apk 100% |********************************| 10888  0:00:00 ETA
'git-secret-0.5.0.apk' saved
fetch https://dl-cdn.alpinelinux.org/alpine/v3.18/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.18/community/x86_64/APKINDEX.tar.gz
ERROR: git-secret-0.5.0.apk: IO ERROR
```

**Comment by sobolevn** [comment]
Yes, for some reason they now ask me to upgrade to a paid account. I've reached them to activate their OS plan. 

Sorry for this! Packaging is hard :(

**Comment by FanchenBao** [comment]
Access to jfrog is working on our pipeline now.

**Comment by Morishiri** [comment]
@sobolevn seems we have this issue again :(

**Comment by sobolevn** [comment]
Trying to reactivate, but looks like JFrog is down for me.

**Comment by steveonjava** [comment]
I asked JFrog ops to re-enable the server for you.

We got the OSS sponsorship request and it is approved, but the e-mail chain got dropped. I replied to it just now, so please acknowledge you would like to officially join our OSS sponsored program.

**Comment by sobolevn** [comment]
Thank you!

**Comment by steveonjava** [comment]
The server is re-enabled now. However, it may get deactivated since it is a limited time free instance. I sent you instructions via e-mail on how to request a new server instance that won't expire. (if you have trouble receiving the e-mail, please let me know)

**Comment by sobolevn** [comment]
I will do this as soon as I will get home from my trip (in a week). Thanks!

**Comment by joshrabinowitz** [comment]
@sobolevn did this get resolved?

**Comment by sobolevn** [comment]
yes, I think so!
